Source/WebCore/platform/gtk/DragImageGtk.cpp

```diff
@@ -21,13 +21,21 @@
 
 #include "Element.h"
 #include "Image.h"
-#include "NotImplemented.h"
 #include "TextFlags.h"
 #include "TextIndicator.h"
 #include <cairo.h>
 #include <gdk/gdk.h>
 #include <wtf/URL.h>
 
+#if USE(SKIA)
+IGNORE_CLANG_WARNINGS_BEGIN("cast-align")
+#include <skia/core/SkBitmap.h>
+#include <skia/core/SkCanvas.h>
+#include <skia/core/SkImageInfo.h>
+#include <skia/core/SkPixmap.h>
+IGNORE_CLANG_WARNINGS_END
```

Comment:
It also gets pulled in by `SkBitmap.h`.

Annoying the style checker really doesn't like:

```c
#if USE(SKIA)
#include <skia/core/SkCanvas.h>
#include <skia/core/SkImageInfo.h>

IGNORE_CLANG_WARNINGS_BEGIN("cast-align")
#include <skia/core/SkBitmap.h>
#include <skia/core/SkPixmap.h>
IGNORE_CLANG_WARNINGS_END
#endif
``

So it would be something ugly like:

```c
#if USE(SKIA)
#include <skia/core/SkCanvas.h>
#include <skia/core/SkImageInfo.h>
#endif

#if USE(SKIA)
IGNORE_CLANG_WARNINGS_BEGIN("cast-align")
#include <skia/core/SkBitmap.h>
#include <skia/core/SkPixmap.h>
IGNORE_CLANG_WARNINGS_END
#endif
```